querysel.js

```diff
@@ -104,3 +104,16 @@ export const blurHandler = (e) => {
     show(classes, 'success', 'error');
   }
 };
+
+/**
+ * @function autoFIll Handles the auto filling of the inputs with available data in local storage
+ */
+export const autoFill = () => {
+  const data = JSON.parse(localStorage.getItem('data'));
+  if (data) {
+    const { name, email, message } = data;
+    inputs[0].value = name;
+    inputs[1].value = email;
+    inputs[2].value = message;
+  }
+};
```

Comment:
- [ ] Please note that on page reload all data that was filled in the form was lost. Kindly make sure that all data is retained tafter page reload😉. You can use` localStorage.setItem('contactFormInput', JSON.stringify(formData));` to store the data to localStrogae👌